src/index.css

```diff
@@ -104,6 +105,70 @@ h1 {
   margin-top: 8px;
 }
 
+nav {
+  background-color: #262626;
+  width: 100%;
+  position: sticky;
+  top: 0;
+  height: auto;
+}
+
+nav li {
+  font-size: 19px;
+  font-weight: 700;
+  line-height: 26px;
+  text-align: center;
+  margin-right: 16px;
+  margin-bottom: 8px;
+  margin-left: 16px;
+  padding: 8px 16px 8px 16px;
+}
+
+nav li:hover {
+  background-color: #797876;
+}
+
+nav ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: auto;
+  padding-bottom: 66px;
+  padding-top: 30px;
+}
+
+nav a {
+  text-decoration: none;
+  color: white;
+}
+
+nav a.active-link {
+  background-color: #ffffff;
+  color: black;
+}
+nav ul li:has(a.active-link) {
+  background-color: white;
```

Comment:
Sometimes you're using `#ffffff` and sometimes `white`. At some point you could introduce color variables to ensure a consistent approach to coloring. But this is just a note for the future. No need to refactor color organization in this PR.